test/sql-tap/default.test.lua

```diff
@@ -91,7 +91,7 @@ test:do_execsql_test(
 	PRAGMA table_info(t4);
 	]], {
 	-- <default-2.2>
-	0,"ROWID","integer",1,"",1,1,"C","string",0,"'abc'",0
+    0,"ROWID","integer",1,"",1,1,"C","string",0,"",0
```

Comment:
AFAIU, there is a bug in `checkpatch.pl` or `luacheck` forcing @ImeevMA to follow Lua guidelines (i.e. 4 spaces instead of tabs) in scope of the changed lines (please correct me if I'm wrong). At least I asked about the same issues in #9249. Anyway, I guess it's more convenient to adjust all SQL tests at once in a separate PR.

Reply:
checkpatch.pl does not allow usage of tabs, so in changed tests I use 4 spaces instead. I am not sure why 4 spaces, but I believe I saw rule about this.